mindmeld/active_learning/classifiers.py

```diff
@@ -168,9 +172,99 @@ def _validate_class_level_statistic(class_level_statistic):
             )
         return class_level_statistic
 
+    @staticmethod
+    def _get_entity_vector(
```

Comment:
Is this method used anywhere?

Reply:
Removed (has been moved to label map class)